runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcWindmillStreamFactory.java

```diff
@@ -191,63 +205,18 @@ public void appendSummaryHtml(PrintWriter writer) {
     }
   }
 
-  static class Builder {
-    private static final int NO_HEALTH_CHECK = -1;
-    private final JobHeader jobHeader;
-    private int logEveryNStreamFailures;
-    private int streamingRpcBatchLimit;
-    private int windmillMessagesBetweenIsReadyChecks;
-    private Supplier<Duration> maxBackOff;
-    private int healthCheckInterval;
-
-    private Builder(JobHeader jobHeader) {
-      this.jobHeader = jobHeader;
-      this.logEveryNStreamFailures = DEFAULT_LOG_EVERY_N_STREAM_FAILURES;
-      this.streamingRpcBatchLimit = DEFAULT_STREAMING_RPC_BATCH_LIMIT;
-      this.windmillMessagesBetweenIsReadyChecks = DEFAULT_WINDMILL_MESSAGES_BETWEEN_IS_READY_CHECKS;
-      this.maxBackOff = () -> DEFAULT_MAX_BACKOFF;
-      this.healthCheckInterval = NO_HEALTH_CHECK;
-    }
+  @AutoBuilder(ofClass = GrpcWindmillStreamFactory.class)
```

Comment:
used AutoBuilder to reduce boiler blate
changed the gradle file to ignore checker framework for generated files (other directories that have generated files have this as well.)